LocalPayment/src/main/java/com/braintreepayments/api/LocalPaymentRequest.java

```diff
@@ -45,6 +45,28 @@ public class LocalPaymentRequest {
     private String phone;
     private boolean shippingAddressRequired;
     private String surname;
+    private final boolean hasUserLocationConsent;
+
+    /**
+     * Deprecated. Use {@link LocalPaymentRequest#LocalPaymentRequest(boolean)} instead.
+     **/
+    @Deprecated
+    public LocalPaymentRequest() {
+        hasUserLocationConsent = false;
```

Comment:
nitpick - should we call the new constructor with (false) here? Just to future proof for any other logic that could be added to the constructor

Reply:
Great suggestion. let me update this